mlrun/feature_store/api.py

```diff
@@ -854,11 +854,14 @@ def _ingest_with_spark(
         featureset.update_targets_for_ingest(targets_to_ingest, overwrite=overwrite)
 
         for target in targets_to_ingest or []:
+            wrong_path = False
             if type(target) is DataTargetBase:
                 target = get_target_driver(target, featureset)
+            elif type(target) is not DataTargetBase and target.path is None:
+                wrong_path = True
             if featureset.spec.passthrough and target.is_offline:
                 continue
-            if target.path and urlparse(target.path).scheme == "":
+            if (target.path and urlparse(target.path).scheme == "") or wrong_path:
```

Comment:
Why do we want to raise `Paths for spark ingest must contain schema, i.e v3io, s3, az` when the target class is not what we expected? That doesn't seem to make sense.

Reply:
I don't really understand what you mean... we do get class that we expected just the path isn't much the schema we want..

Reply:
Note:
The main aim is:
 - in case in Source e.g. Pandas Data Frame and engine Spark, the Target can be mlrun.datastore.ParquetTarget()
 - it means if you need s3, ... you have to change default path in case of engine Spark